patches/chrome-tools-build-win-create_installer_archive.py.patch

```diff
@@ -1,92 +1,33 @@
-+  locales_dest_path = staging_dir
-+  locales_dest_path = os.path.join(locales_dest_path, config.get('GENERAL', 'brave_resources.pak'),
-+    'resources', 'brave_extension', '_locales')
-+  locales_dest_path = os.path.realpath(locales_dest_path)
-+  try:
-+    shutil.rmtree(locales_dest_path)
-+  except:
-+    pass
-+  shutil.copytree(locales_src_dir_path, locales_dest_path)
-+  # Files are copied, but we need to inform g_archive_inputs about that
-+  for root, dirs, files in os.walk(locales_dest_path):
-+    for name in files:
-+        rel_dir = os.path.relpath(root, locales_dest_path)
-+        rel_file = os.path.join(rel_dir, name)
-+        candidate = os.path.join('.', 'resources', 'brave_extension' '_locales', rel_file)
-+        g_archive_inputs.append(candidate)
-+
-+def CopyBraveRewardsExtensionLocalization(config, staging_dir):
-+  """Copies extension localization files from
-+  \\brave\components\brave_rewards\extension\brave_rewards\_locales to
-+  \\<out_gen_dir>\chrome\installer\mini_installer\mini_installer\temp_installer_archive\Chrome-bin\<version>\resources\brave_rewards\_locales
-+  """
-+  brave_extension_dir = os.path.realpath(os.path.dirname(os.path.realpath(__file__)) )
-+  brave_extension_dir = os.path.realpath(os.path.join(brave_extension_dir,
-+    os.pardir,  os.pardir, os.pardir, os.pardir, 'brave', 'components', 'brave_rewards', 'resources', 'extension', 'brave_rewards'))
-+  locales_src_dir_path = os.path.join(brave_extension_dir, '_locales')
-+  locales_dest_path = staging_dir
-+  locales_dest_path = os.path.join(locales_dest_path, config.get('GENERAL', 'brave_resources.pak'),
-+    'resources', 'brave_rewards', '_locales')
-+  locales_dest_path = os.path.realpath(locales_dest_path)
-+  try:
-+    shutil.rmtree(locales_dest_path)
-+  except:
-+    pass
-+  shutil.copytree(locales_src_dir_path, locales_dest_path)
-+  # Files are copied, but we need to inform g_archive_inputs about that
-+  for root, dirs, files in os.walk(locales_dest_path):
-+    for name in files:
-+        rel_dir = os.path.relpath(root, locales_dest_path)
-+        rel_file = os.path.join(rel_dir, name)
-+        candidate = os.path.join('.', 'resources', 'brave_rewards' '_locales', rel_file)
-+        g_archive_inputs.append(candidate)
-+
-+def CopyPreSignedBinaries(output_dir, staging_dir, current_version):
-+  """Copies already signed three binaries - brave.exe, chrome.dll and chrome_child.dll
-+  These files are signed during the build phase to create widevine sig files.
-+  """
-+  src_dir = os.path.join(output_dir, 'signed_binaries')
-+  chrome_dir = os.path.join(staging_dir, CHROME_DIR)
-+  version_dir = os.path.join(chrome_dir, current_version)
-+  shutil.copy(os.path.join(src_dir, 'brave.exe'), chrome_dir)
-+  shutil.copy(os.path.join(src_dir, 'chrome.dll'), version_dir)
-+  shutil.copy(os.path.join(src_dir, 'chrome_child.dll'), version_dir)
++  from create_installer_archive_helper import (
```

Comment:
please put on one line, lint is not an issue for patches

Reply:
done.